BParkingNano/python/electronsBPark_cff.py

```diff
@@ -53,6 +52,9 @@
 )
 egmGsfElectronIDSequence = cms.Sequence(egmGsfElectronIDTask)
 
+# regression stuff
+from RecoEgamma.EgammaTools.regressionModifierNew_cfi import regressionModifier106XUL
+from RecoEgamma.EgammaTools.regressionModifierNew_cfi import regressionModifier106XULLP
```

Comment:
are both needed?

Reply:
@cavalari

Reply:
yes, one is for the PF ele regression and the other is for the LP ele regression. The PF ele regression is simple, and uses the same variables as the standard EGM regression, while the LP ele regression uses a few extra variables relative to the 1st sub-cluster of the supercluster.